code/game/objects/items/weapons/towels.dm

```diff
@@ -93,17 +96,17 @@
 
 /obj/item/towel/proc/dry_mob(mob/living/target, mob/living/user)
 	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
-	var/reagent_space = reagents.maximum_volume - reagents.total_volume
+	var/reagent_space = REAGENT_MAXIMUM_VOLUME(reagents) - REAGENT_TOTAL_VOLUME(reagents)
```

Comment:
not sure why i didn't use REAGENTS_FREE_SPACE here, but now that would make it faster i think, since it'd go from 2 typechecks to 1